eq.js

```diff
@@ -14,16 +14,25 @@ const __ = require('./_internal/placeholder')
  * var value any,
  *   leftCompare any,
  *   rightCompare any,
- *   left (value=>Promise|leftCompare)|leftCompare,
- *   right (value=>Promise|rightCompare)|rightCompare
+ *   left value|(value=>Promise|leftCompare)|leftCompare,
+ *   right value|(value=>Promise|rightCompare)|rightCompare
+ *
+ * eq(left, right) -> boolean
  *
  * eq(left, right)(value) -> Promise|boolean
+ *
  * ```
  *
  * @description
  * Test for [SameValueZero](http://ecma-international.org/ecma-262/7.0/#sec-samevaluezero) between the returns of two functions. Either parameter may be an actual value for comparison.
  *
  * ```javascript [playground]
+ * const isNamesEqual = eq('Ted', 'George')
+ *
+ * console.log(isNamesEqual) // false
+ * ```
+ *
```

Comment:
let's add in between here
```
If both arguments are functions, `eq` treats those functions as argument resolvers and returns a function that first resolves its arguments by the argument resolvers before making the comparison.

If only one argument is a function, `eq` still returns a function that resolves its arguments by the argument resolver, treating the value (non function) argument as an already resolved value for comparison.
```

could we also use this text for gt, gte, lt, and lte? All you would have to do is replace the `eq` part with the appropriate function name